articles/machine-learning/resource-azure-container-for-pytorch.md

```diff
@@ -43,14 +43,13 @@ Azure Container for PyTorch is a lightweight, standalone environment that includ
 
 The following configurations are supported:
 
-| Environment Name | OS | GPU Version| Python Version | PyTorch Version | ORT-training Version | DeepSpeed Version | torch-ort Version |
-| --- | --- | --- | --- | --- | --- | --- | --- |
-|acpt-pytorch-2.0-cuda11.7|Ubuntu 20.04|cu117|3.8|2.0|1.14.1|0.8.2 |0.14.0|
-|acpt-pytorch-1.13-cuda11.7|Ubuntu 20.04|cu117|3.8|1.13.1|1.14.1|0.8.2|1.14.0|
-|acpt-pytorch-1.12-py39-cuda11.6|Ubuntu 20.04|cu116|3.9|1.12.1|1.14.1|0.8.2|1.14.0|
-|acpt-pytorch-1.12-cuda11.6|Ubuntu 20.04|cu116|3.8|1.12.1|1.14.1|0.8.2|1.14.0|
-|acpt-pytorch-1.11-cuda11.5|Ubuntu 20.04|cu115|3.8|1.11.0|1.11.1|0.7.3|1.11.0|
```

Comment:
acpt-pytorch-1.11-cuda11.5 should not be removed as it's still present in AML. It just has no version change

Reply:
I couldn't find it in my Azure ML workspace. Could you double check if _acpt-pytorch-1.11-cuda11.5_ exists or not?

<img width="1376" alt="image" src="https://github.com/MicrosoftDocs/azure-docs/assets/3815677/d4a9b3b4-8879-4a68-845e-508418618c55">

Reply:
I can't see it to but we haven't got a deprecation notice. I will confirm with the AML team but let's go ahead with this PR. We can add cu11.5 later if it shows up